src/test/resources/features/Section4/Standard/StandardJourney.feature

```diff
@@ -32,7 +41,84 @@ Feature: Section4 Standard Prelodged
     And I check the MiniCYA page for Section-4
     And I click continue on MiniCya
 
+    # validate dynamic title change on previous documents page
+    # check title on previous documents page when authorisation choice is Permanent with excise
+    And I navigate to Authorisation Choice page
+    And I select Permanent with excise as export procedure choice
+    And I click continue
+    And I navigate to Previous Documents page
+    Then I should land on Previous-Document page
```

Comment:
It depends on the navigation we need to see what pages are navigating to and how it looks like in summary section.

Reply:
explained this to siva over slack